api/external.go

```diff
@@ -286,7 +289,11 @@ func (a *API) internalExternalProviderCallback(w http.ResponseWriter, r *http.Re
 	rurl := a.getExternalRedirectURL(r)
 	if token != nil {
 		q := url.Values{}
-		q.Set("provider_token", providerToken)
+		q.Set("provider_token", providerAccessToken)
+		// Because not all providers give out a refresh token
+		if providerRefreshToken != "" {
+			q.Set("provider_refresh_token", providerRefreshToken)
+		}
```

Comment:
can we add a link as a comment to the oauth2 spec here? (https://www.rfc-editor.org/rfc/rfc6749.html#section-5.1)

Reply:
Sure 👍 ! I added it in this commit:  https://github.com/supabase/gotrue/pull/641/commits/30ffbbb736864c48fa690c1d0c275bdf8aedb39b